### PR TITLE
Revert the previous change in 250681196f

### PR DESCRIPTION
**Pull request by aparent**
Make style check fail when astyle fails.

### CODE REVIEW

**Comment by aparent** [comment]
Looking at the build output more closely it also seems like the error "find: unknown predicate `-E'" has been happening since e937a6a664df755.

**Comment by aparent** [comment]
Will merge this later today if there are no objections.

File: .travis-style-check.sh [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+	tput sgr 0;
+	exit 1;
+elif [ ${STATUS[0]} != 0 ];  then
+	tput setaf 1;
+	echo "prettyprint failed.";
+	tput sgr 0;
 	exit 1;
 else 
 	tput setaf 2;
 	echo "Code adheres to the project standards.";
```
Comment (tlepoint) [inline .travis-style-check.sh]: Maybe we should add `Code adheres to the project standards (prettyprint)` here.

File: .travis-global-namespace-check.sh [hunk 2]
```diff
@@ -4,10 +4,12 @@ if [[ $(nm -g liboqs.a | grep ' T ' | grep -E -v -i ' T [_]?OQS') ]];
 then 
 	tput setaf 1;
 	echo "Code contains the following non-namespaced global symbols; see https://github.com/open-quantum-safe/liboqs/wiki/Coding-conventions for function naming conventions.";
+	tput sgr 0
 	nm -g liboqs.a | grep ' T ' | grep -E -v -i ' T [_]?OQS'
 	exit 1;
 else 
 	tput setaf 2;
 	echo "Code adheres to the project standards.";
```
Comment (tlepoint) [inline .travis-global-namespace-check.sh]: Maybe we should add `Code adheres to the project standards (global namespace)` here.

**Comment by aparent** [comment]
Good idea, Added.

**Pull request merged**